LICENSE.txt

```diff
@@ -6,34 +6,39 @@ Most of the Checker Framework is licensed under the GNU General Public
 License, version 2 (GPL2), with the classpath exception.  The text of this
 license appears below.  This is the same license used for OpenJDK.
 
-A few parts of the Checker Framework have more permissive licenses.
+A few parts of the Checker Framework have more permissive licenses, notably
+the parts that you might want to include with your own program.
 
  * The annotations and utility files are licensed under the MIT License.
-   (The text of this license also appears below.)  More specifically, all
-   the parts of the Checker Framework that you might want to include with
-   your own program use the MIT License.  This is the checker-qual*.jar and
-   checker-compat-qual*.jar files and all the files that appear in them:
-   every file in a qual/ directory, plus utility files such as
-   NullnessUtil.java, RegexUtil.java, SignednessUtil.java, etc.  In
-   addition, the cleanroom implementations of third-party annotations,
-   which the Checker Framework recognizes as aliases for its own
-   annotations, are licensed under the MIT License.
-
-Some external libraries that are included with the Checker Framework have
-different licenses.
+   (The text of this license also appears below.)  This applies to the
+   checker-qual*.jar and all the files that appear in it:  every file in a
+   qual/ directory, plus utility files including FormatUtil.java,
```

Comment:
I recommend not using the word "including" because that implies that there are additional files that meet this criterion that are not listed. Just dropping that one word will make this perfect.

Reply:
The reason I put "including" is that I'm worried that someone will add a new file without updating this list.  The general description is correct, and the list is correct at the moment.  I'll remove it nonetheless and will hope that it doesn't get out of date in the future.

Reply:
Thank you! I know it seems nit-picky but it really helps downstream users a lot!